### PR TITLE
[Word] (DocumentCreated) Add code examples

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Related issue: https://github.com/OfficeDev/office-js-docs-reference/issues/1200

### CODE REVIEW

**Comment by ElizabethSamuel-MSFT** [comment]
Feature crew has signed off

File: docs/code-snippets/word-snippets.yaml [hunk 1]
```diff
@@ -1,3 +1,19 @@
+Word.Application#createDocument:member(1):
+  - |-
+    // Update the text of the current document using the text from another document passed in as base64.
```
Comment (AlexJerabek) [inline docs/code-snippets/word-snippets.yaml:3]:
```suggestion
    // Update the text of the current document with the text from another document passed in as a base64-encoded string.
```

File: docs/code-snippets/word-snippets.yaml [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+    await Word.run(async (context) => {
+        const externalDoc = context.application.createDocument(getBase64());
```
Comment (AlexJerabek) [inline docs/code-snippets/word-snippets.yaml:5]: Where does `getBase64` come from? That feels like an important step.

File: docs/code-snippets/word-snippets.yaml [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+        await context.sync();
+
+        const externalDocBody = externalDoc.body;
+        externalDocBody.load("text");
+        await context.sync();
+
+        const externalDocBodyText = externalDocBody.text;
+        const body = context.document.body;
+        body.insertText(externalDocBodyText, WordInsertLocation.start);
```
Comment (AlexJerabek) [inline docs/code-snippets/word-snippets.yaml:14]: Consider this naming suggestion to help delineate between the two bodies.
```suggestion
        const currentDocBody = context.document.body;
        currentDocBody .insertText(externalDocBodyText, WordInsertLocation.start);
```

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
I left a few comments. They apply to both samples. My main question is what would it take to add the `getBase64` method to show how someone gets that info?

File: docs/code-snippets/word-snippets.yaml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/code-snippets/word-snippets.yaml:5]: Asked code source for this and now waiting for their response

Comment (ElizabethSamuel-MSFT) [reply docs/code-snippets/word-snippets.yaml:8]: In the meantime, added a comment about need to implement getBase64 that's similar to the comment in [Word.Body.insertFileFromBase64](https://learn.microsoft.com/en-US/javascript/api/word/word.body?view=word-js-preview#word-word-body-insertfilefrombase64-member(1))

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**